### DISCUSSION TITLE
Codespaces doesn't respect "Remote: Local Forward Port" vscode setting

### DISCUSSION THREAD

**Discussion by ukalwa** [community/community#30089] [Codespaces]
In vscode settings, we can set local forward ports to `localhost` and `allInterfaces`. `localhost` forwards the ports on localhost while the other option forwards ports on `0.0.0.0`. Codespaces doesn't seem to respect this vscode setting.

I have restarted vscode and tried manually changing local address to `0.0.0.0` but none of them worked.

**Comment by jfullstackdev** [comment]
did you try manually forwarding it on web Codespaces ?

**Reply by ukalwa** [reply]
I have not tried it on web codespaces. I have had issues with port forwarding on web codespaces due to company proxy so I always prefer vscode client to connect to it.

I see `127.0.0.1:5000` but I want to expose the port on all interfaces on my computer
![image](https://user-images.githubusercontent.com/11986128/185697473-b1e990cb-82e5-4fde-8bfe-b3dbacc819e7.png)
I added `"remote.localPortHost": "allInterfaces"` this in my settings.json. Also, in the "Port visibility" menu, I only see Private and Private to organization.

**Reply by jfullstackdev** [reply]
is it synced ? did you install the extension ? 

https://code.visualstudio.com/docs/remote/codespaces

![image](https://user-images.githubusercontent.com/47092464/185750903-42542ec6-7168-4e84-886c-480701ad53e7.png)

**Reply by ukalwa** [reply]
Yes, it is synced and the extension is installed. But it looks like I can forward ports using [gh cli](https://cli.github.com/manual/gh_codespace_ports_forward). So, I will probably just use that for my usecase. Thanks for the help 😄
